### DISCUSSION TITLE
[Feature] Integration of LAION-Aesthetics model

### DISCUSSION THREAD

**Discussion by Ben29030** [immich-app/immich#11217] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I would like to suggest the integration of the LAION-Aesthetics model as an additional machine learning option within Immich, which could provide several valuable functionalities:

Duplicate Management: Use the model to select the best-looking image when managing duplicates.

Slideshows: Improve "X-year ago" slideshows by choosing the most aesthetically pleasing photos.

Search Function and filtering: Enhance the search function with an option to find images based on their aesthetic. 

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#7202
